### PR TITLE
Updates to breaking-changes.md

### PR DESCRIPTION
**Pull request by ajcvickers**
Specifically:
* New note for https://github.com/aspnet/EntityFrameworkCore/issues/13573
* First attempt at a summary--feedback appreciated. Not sure if a table is right here?

Edit by @Rick-Anderson  [Internal Review URL](https://review.docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.0/breaking-changes?branch=pr-en-us-1585)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ajcvickers  I'm fixing all the broken `#in-page-bookmarks` in #1586
Next time you have a page like this, I can edit the page and add the bookmarks.
cc @tdykstra

**Comment by divega** [comment]
@ajcvickers I would like to see how the table looks like in the review page, but apparently it is not being rendered. 

It is possible that something is broken in the build (I sent email asking for help), but the build is also complaining about numerous invalid links in this particular PR. For example, `#The-Relational:TypeMapping-annotation-is-now-just-TypeMapping` should be `#the-relationaltypemapping-annotation-is-now-just-typemapping`, so perhaps that is blocking the review page form being published.

**Comment by divega** [comment]
Ah, I missed your message @Rick-Anderson. Let's see how the preview looks like in your PR.

**Comment by divega** [comment]
Reviews have been re-enabled as of this morning so I was finally able to look at what the change looks like. The formatting of the table looks pretty neat and I like having the information about the impact alongside the issue. 

On the other hand, it is hard for me to appreciate the value this adds when I compare it with the clunckiness it introduces in to both the UI (the whole list is now repeated three times) and to the process of updating the documentation page.

Perhaps we should figure out a way to add the impact alongside (or below) individual tittles.

**Comment by Rick-Anderson** [comment]
@ajcvickers @divega  the  [Internal Review URL](https://review.docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.0/breaking-changes?branch=pr-en-us-1585)  page is working.

**Comment by ajcvickers** [comment]
@divega My thoughts after looking at it:
* There is too much here not to have a summary where one can scan a list to find what's here
* The summary created to the right is useless--it's just a squashed up mess
  * If we could get it not to render, then great, but I doubt that is possible
  * I can't think of any way to make it readable without making the contents so terse that they are useful
  * So mentally, I'm just abandoning it as an unfortunate U.I. implementation
* I can't currently think of a different way of putting the summary at the top, but I'm open to ideas.

Bottom line for me:
* There needs to be an effective summary
* The panel at the right is useless for this (in this case--it does work somewhat in other docs, although I think it looks pretty bad most of the time.)
* I can't currently think of a better way to do the summary.

**Comment by Rick-Anderson** [comment]
> The summary created to the right is useless--it's just a squashed up mess

I can get rid of that

**Comment by ajcvickers** [comment]
@Rick-Anderson Great!

**Comment by Rick-Anderson** [comment]
@ajcvickers approve  #1597 - see [Internal review URL](https://review.docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.0/breaking-changes?branch=pr-en-us-1597) which removes everything but summary. Lef me know if I can S&M into your branch.

**Comment by ajcvickers** [comment]
@Rick-Anderson That looks pretty good to me.
@divega Thoughts?

**Comment by divega** [comment]
@ajcvickers, I agree that now it looks pretty good. Thanks @Rick-Anderson, nice hack! Now that we overcame the main issue, I will finish the review.

File: entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md [hunk 1]
```diff
@@ -46,6 +98,7 @@ Besides this, automatic client evaluation can lead to issues in which improving
 
 If a query can't be fully translated, then either rewrite the query in a form that can be translated, or use `AsEnumerable()`, `ToList()`, or similar to explicitly bring data back to the client where it can then be further processed using LINQ-to-Objects.
 
+<a name="no-longer"></a>
```
[the remark this answers is not in the document]
Comment (ajcvickers) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: Agreed. @Rick-Anderson Can we get short handles here without it breaking existing links?

[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: > Can we get short handles here without it breaking existing links?

Most of the links were broken, I shortened them to fix them.  If you can revert to before I made any changes, I can fix only the broken links.

[the remark this answers is not in the document]
Comment (ajcvickers) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: @Rick-Anderson So you're saying most of the external links that we already have to sections of this document were broken before I made this change? How did that happen? What did we do wrong? (Also, all the ones I can find do work, so can you point to which ones are broken?)

[the remark this answers is not in the document]
Comment (divega) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: @ajcvickers I think what @Rick-Anderson means is that some of the links where broken in your original PR and when he decided to fix them, he also decided to shorten then.

[the remark this answers is not in the document]
Comment (divega) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: So the target links were ok, only some of the links you included in the table when you originally put it together were incorrect.

[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: > your original PR and when he decided to fix them, he also decided to shorten then.

Yes, I shortened them to make them work.  They looked like they were an attempt to duplicate the automatic generated links. If that's the case, the auto-generated links should still work.

@tdykstra  Going from H2 to H3 doesn't changes the links does it? H3 links aren't added to the TOC on the right, that's why I made that change.

[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: @ajcvickers See #1586

There were so many broken links I assumed most of them were broken. It was in the build report.

[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: If you can revert my commit, I'll fix only the broken links.

[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: My change didn't change any of the existing links, it only added new links.

[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: The links are created automatically by the H2 or H3. As long as you don't change the title, the link doesn't change. The bookmark references in the summary table were mostly broken. So I manually added an additional link and used that in the table.

[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: view https://review.docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.0/breaking-changes?branch=pr-en-us-1597 and hover over any of the H3's - That link didn't change. I just added an additional link on top

[the remark this answers is not in the document]
Comment (ajcvickers) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: Ah, okay. Then I think we're good, if @divega agrees.

[the remark this answers is not in the document]
Comment (divega) [reply entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md]: Yes, this is fine then.
